src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -779,31 +767,32 @@ def test_join_leave_join(self):
         )
 
         # capture the invite code from stdout
-        invite_code = outcome.stdout.strip()
+        invite_code = outcome.stdout.strip().encode("utf8")
```

Comment:
Seems weird outcome would be unicode here.  Some code probably accidentally wrote a unicode string to stdout somewhere and it implicitly (ascii) decoded all of outcome.stdout to unicode?

Reply:
(code is now gone)